imports/api/accounts/methods.js

```diff
@@ -85,8 +85,15 @@ Meteor.methods({
 
     },
 
-});
+    'insertDefaultCurrency' : function(id){
+        if(!id){
+            throw new Meteor.Error("no", "no default account");
+        }
```

Comment:
`Meteor.Error("no", "no default account")`, I think this is currency, not account.